content/lactf_2024/closed.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Closed
+date: 2024-02-18T20:03:33+02:00
+description: Writeup for Closed [Lactf 2024]
+author: H0N3YP0T
+tags:
+- OSINT
+draft: false
+---
+___
+
+## Challenge Description
+
+Over spring break, my friend sent me this picture of a place they went to, and said it was their favorite plate to visit but it closed :(.
```

Comment:
``plate`` --> ``place``

Reply:
> `plate` --> `place`

Nop it comes from the offcial chall desc :)